components/CodeEditor/CodeEditorContainer/ProblemView.tsx

```diff
@@ -35,6 +50,21 @@ export const ProblemView = ({ problemTitle, problemStatement }: Props) => {
           {problemTitle}
         </Typography>
         <Markdown markdownString={problemStatement} />
+        {problemAuthor && problemDifficulty && problemLastModified ? (
```

Comment:
Should we be checking these all at once or individually? If problemLastModified or another is not available, but another is, then none will be displayed

Reply:
Also going to turn these into badges in restyling to show at the top of the problem